doc/radosgw/qat-accel.rst

```diff
@@ -69,26 +69,82 @@ hardware is available and QAT is capable to handle it.
 
 Configuration
 =============
-1. QAT based Encryption for RGW 
-
-Edit the Ceph configuration file to make use of QAT based crypto plugin::
+1. Prerequisites
+
+Make sure the QAT driver with version v1.7.L.4.14.0 or higher has
+been installed. Remember to set an environment variable "ICP_ROOT" for your QAT
+driver package root directory. 
+
+To enable the QAT based encryption and compression, user needs to modify the QAT
+configuration files. For example, for Intel QuickAssist Adapter 8970 product, revise 
+c6xx_dev0/1/2.conf in the directory ``/etc/`` and keep them the same, e.g.::
+
+    …
+    ##############################################
+    # User Process Instance Section
+    ##############################################
+    [CEPH]
+    NumberCyInstances = 1
+    NumberDcInstances = 1
+    NumProcesses = 8
+    LimitDevAccess = 1
+    
+    # Crypto - User instance #0
+    Cy0Name = "SSL0"
+    Cy0IsPolled = 1
+    # List of core affinities
+    Cy0CoreAffinity = 0
+    
+    # Data Compression - User instance #0
+    Dc0Name = "Dc0"
+    Dc0IsPolled = 1
+    # List of core affinities
+    Dc0CoreAffinity = 0
+
+
+2. QAT based Encryption for RGW 
+
+The CMake option ``WITH_QAT=ON`` must be configured. If you `build Ceph`_ source code,
+navigate to your cloned Ceph repository and execute the following::
```

Comment:
please use the `.. prompt:: bash $` directive.

Reply:
Sure, I have add this directive.